admin-ui/src/pages/request-detail-page.tsx

```
@@ -132,13 +135,13 @@ export function RequestDetailPage(): React.JSX.Element {
     return (
       <Card>
         <CardHeader>
-          <CardTitle>Request</CardTitle>
-          <CardDescription>Request not found.</CardDescription>
+          <CardTitle>{t("requestDetailPage.title")}</CardTitle>
+          <CardDescription>{t("requestDetailPage.notFoundDescription")}</CardDescription>
```

ESLint

[error] Replace `{t("requestDetailPage.notFoundDescription")}` with `⏎············{t("requestDetailPage.notFoundDescription")}⏎··········`

(prettier/prettier)